Documentation/content/docs/develop_build.md

```diff
@@ -7,9 +7,11 @@ The library can be built with webpack automatically. Webpack can either gather a
 
 ## Building vtk.js
 
+Install dependencies by running `npm install`.
+
 In order to build the library you can run `npm run dev` for quick development usage or `npm run build` for production usage.
 
-Either of these commands will generate a `dist/vtk.js` file that can then be used as an external script.
+These commands will generate an ES module in `dist/esm`.
```

Comment:
@floryst is that true ? Or do we have a post-script for generating the ESM?

Reply:
These currently aren't correct, but my plan is to make it as such. I think we need to have some changes to the build, namely integrating the post-script into the build step. The post-script simply makes it a proper package, among other fix-ups.

@ebrahimebrahim: for now, I would suggest the following:
- Change `npm run dev` to `npm run dev:esm` and `npm run build` to `npm run build:esm`. Those two will generate the ES module.

Reply:
Great, I just made that change. Currently `npm run build:esm` works but there is no script for `dev:esm`.